connectivity/drivers/cellular/UBLOX/AT/UBLOX_AT_CellularStack.cpp

```diff
@@ -372,7 +371,7 @@ UBLOX_AT_CellularStack::CellularSocket *UBLOX_AT_CellularStack::find_socket(int
 {
     CellularSocket *socket = NULL;
 
-    for (unsigned int x = 0; (socket == NULL) && (x < _device.get_property(AT_CellularDevice::PROPERTY_SOCKET_COUNT)); x++) {
+    for (intptr_t x = 0; (socket == NULL) && (x < _device.get_property(AT_CellularDevice::PROPERTY_SOCKET_COUNT)); x++) {
```

Comment:
`intptr_t`? That's not what you want here. (It's for storing pointers as integers).

 You want `size_t` or `ptrdiff_t`, depending on whether you need it to be signed or not.

Reply:
i changed it accordingly to your suggestions